orbitos-amd64-hvm.yml

```diff
@@ -86,3 +86,6 @@ plugins:
   debconf: >-
     # Make kexec-tools handle reboots
     kexec-tools     kexec-tools/load_kexec  boolean true
+  ec2_publish:
+    manifest_url: https://pebble-amis.s3-us-west-2.amazonaws.com/orbit-os/latest.json
```

Comment:
Not yet (it will be once this is merged).

This uses the "ugly" url because it needs to know which bucket/path to upload to.